From: Benedikt Spranger <b.spranger@pengutronix.de>
Date: Tue, 22 Nov 2011 15:19:52 +0100
Subject: [PATCH] fix static declaration

GCC 4.x declares non-static declaration and static definition and vice
versa as error.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>
---
 netboot/main.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/netboot/main.c b/netboot/main.c
index 82759b6..abd9212 100644
--- a/netboot/main.c
+++ b/netboot/main.c
@@ -54,9 +54,9 @@ struct rom_info rom;
 
 static int vendorext_isvalid;
 static unsigned long netmask;
-static struct bootpd_t bootp_data;
+struct bootpd_t bootp_data;
 static unsigned long xid;
-static unsigned char *end_of_rfc1533 = NULL;
+unsigned char *end_of_rfc1533 = NULL;
 
 #ifndef	NO_DHCP_SUPPORT
 #endif /* NO_DHCP_SUPPORT */
